wiiu/input/wiiu_hid.c

```diff
@@ -613,6 +696,17 @@ static void delete_hidclient(HIDClient *client)
       free(client);
 }
 
+static void init_cachealigned_buffer(int32_t min_size, uint8_t **out_buf_ptr, int32_t *actual_size)
+{
+   int cacheblocks = (min_size < 32) ? 1 : min_size / 32;
+   if(min_size > 32 && min_size % 32 != 0)
+      cacheblocks++;
+
+   *actual_size = 32 * cacheblocks;
```

Comment:
`*actual_size = (*actual_size + 0x1F) & ~ 0x1F;` will align to 32bytes.
actual alignment requirement is 64 though, so use `0x3F` instead.